content-for/test-head-footer.html

```diff
@@ -28,4 +28,13 @@
     height: 100%;
     transform: scale(1);
   }
+
+  /*
+   * Setting duration to zero so that the elements will get transitioned/animated immediately
+   * so that the gradual transition/animation will not violate any of the WCAG guidelines
+   */
+  .axe-running * {
+    transition-duration: 0s !important;
+    animation-duration: 0s !important;
```

Comment:
I'd slightly prefer disabling animations and transitions entirely via `animation: none !important; transition: none !important;` as it's been field tested as reliable in quite a bit of internal apps.

I think the main issue with just disabling the duration is that an element may still require some time to switch to the intended state if using `transition-delay`. This uncertainty goes away if we just disable all constituent properties.

Reply:
I don't have strong preference one over the other. However, as you mentioned, since disabling of animation and transition approach is battle tested, I switched to that approach.